[cpp-redis] ‘sleep_for’ is not a member of ‘std::this_thread’

**Host Environment**
- OS: Kali-linux (WSL2)
- Compiler: gcc version 11.2.0 (Debian 11.2.0-13)

**To Reproduce**
Steps to reproduce the behavior:
`./vcpkg install cpp-redis:x64-linux`

**Failure logs**
``` bash
MY_PATH/cpp_redis-src/sources/core/client.cpp: In member function ‘void cpp_redis::client::sleep_before_next_reconnect_attempt()’:
MY_PATH/cpp_redis-src/sources/core/client.cpp:347:21: error: ‘sleep_for’ is not a member of ‘std::this_thread’
  347 |   std::this_thread::sleep_for(std::chrono::milliseconds(m_reconnect_interval_msecs));
```

**Additional context**
This issue is fixed in original repo by [PR#80](https://github.com/microsoft/vcpkg/pull/23762). However this still exists on vcpkg (cpp-redis:v4.3.1#4)
I created a [pull request](https://github.com/microsoft/vcpkg/pull/23762) try fixing this.

I've just cloned a clean vcpkg and run install again, my kali on wsl2 gives the same error:
``` bash
./vcpkg install cpp-redis
Computing installation plan...
A suitable version of cmake was not found (required v3.22.2). Downloading portable cmake v3.22.2...
Downloading cmake...
  https://github.com/Kitware/CMake/releases/download/v3.22.2/cmake-3.22.2-linux-x86_64.tar.gz -> /home/particleg/vcpkg/downloads/cmake-3.22.2linux-x86_64.tar.gz
Extracting cmake...
The following packages will be built and installed:
    cpp-redis[core]:x64-linux -> 4.3.1#4
  * tacopie[core]:x64-linux -> 3.2.0#3
Additional packages (*) will be modified to complete this operation.
Detecting compiler hash for triplet x64-linux...
A suitable version of ninja was not found (required v1.10.2). Downloading portable ninja v1.10.2...
Downloading ninja...
  https://github.com/ninja-build/ninja/releases/download/v1.10.2/ninja-linux.zip -> /home/particleg/vcpkg/downloads/ninja-linux-1.10.2.zip
Extracting ninja...
Restored 1 packages from /home/particleg/.cache/vcpkg/archives in 28.63 ms. Use --debug to see more details.
Starting package 1/2: tacopie:x64-linux
Installing package tacopie[core]:x64-linux...
Elapsed time for package tacopie:x64-linux: 4.013 ms
Starting package 2/2: cpp-redis:x64-linux
Building package cpp-redis[core]:x64-linux...
-- Downloading https://github.com/cpp-redis/cpp_redis/archive/4.3.1.tar.gz -> cpp-redis-cpp_redis-4.3.1.tar.gz...
-- Extracting source /home/particleg/vcpkg/downloads/cpp-redis-cpp_redis-4.3.1.tar.gz
-- Using source at /home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean
-- Configuring x64-linux-dbg
-- Configuring x64-linux-rel
CMake Warning at scripts/cmake/vcpkg_configure_cmake.cmake:410 (message):
  The following variables are not used in CMakeLists.txt:

      MSVC_RUNTIME_LIBRARY_CONFIG

  Please recheck them and remove the unnecessary options from the
  `vcpkg_configure_cmake` call.

  If these options should still be passed for whatever reason, please use the
  `MAYBE_UNUSED_VARIABLES` argument.
Call Stack (most recent call first):
  ports/cpp-redis/portfile.cmake:25 (vcpkg_configure_cmake)
  scripts/ports.cmake:145 (include)


-- Building x64-linux-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:158 (message):
    Command failed: /home/particleg/vcpkg/downloads/tools/cmake-3.22.2-linux/cmake-3.22.2-linux-x86_64/bin/cmake --build . --config Debug --target install -- -v -j17
    Working Directory: /home/particleg/vcpkg/buildtrees/cpp-redis/x64-linux-dbg
    See logs for more information:
      /home/particleg/vcpkg/buildtrees/cpp-redis/install-x64-linux-dbg-out.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_build_cmake.cmake:106 (vcpkg_execute_build_process)
  scripts/cmake/vcpkg_install_cmake.cmake:44 (vcpkg_build_cmake)
  ports/cpp-redis/portfile.cmake:32 (vcpkg_install_cmake)
  scripts/ports.cmake:145 (include)


Error: Building package cpp-redis:x64-linux failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `git pull` and `./vcpkg update`.
Then check for known issues at:
  https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+cpp-redis
You can submit a new issue at:
  https://github.com/microsoft/vcpkg/issues/new?template=report-package-build-failure.md&title=[cpp-redis]+Build+error
including:
  package: cpp-redis[core]:x64-linux -> 4.3.1#4
    vcpkg-tool version: 2022-03-09-1affd32f93b299d5a907816c328ca3ededb73a7e
    vcpkg-scripts version: a76eb002a 2022-03-24 (9 hours ago)

Additionally, attach any relevant sections from the log files above.
```
The content in `/home/particleg/vcpkg/buildtrees/cpp-redis/install-x64-linux-dbg-out.log`:
``` bash
[1/16] /usr/bin/c++  -I/home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/includes -I/home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/deps/include -I/home/particleg/vcpkg/installed/x64-linux/include -fPIC -std=c++11 -W -Wall -Wextra -O3 -g -fPIC -MD -MT CMakeFiles/cpp_redis.dir/sources/misc/logger.cpp.o -MF CMakeFiles/cpp_redis.dir/sources/misc/logger.cpp.o.d -o CMakeFiles/cpp_redis.dir/sources/misc/logger.cpp.o -c /home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/sources/misc/logger.cpp
[2/16] /usr/bin/c++  -I/home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/includes -I/home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/deps/include -I/home/particleg/vcpkg/installed/x64-linux/include -fPIC -std=c++11 -W -Wall -Wextra -O3 -g -fPIC -MD -MT CMakeFiles/cpp_redis.dir/sources/builders/builders_factory.cpp.o -MF CMakeFiles/cpp_redis.dir/sources/builders/builders_factory.cpp.o.d -o CMakeFiles/cpp_redis.dir/sources/builders/builders_factory.cpp.o -c /home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/sources/builders/builders_factory.cpp
[3/16] /usr/bin/c++  -I/home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/includes -I/home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/deps/include -I/home/particleg/vcpkg/installed/x64-linux/include -fPIC -std=c++11 -W -Wall -Wextra -O3 -g -fPIC -MD -MT CMakeFiles/cpp_redis.dir/sources/builders/error_builder.cpp.o -MF CMakeFiles/cpp_redis.dir/sources/builders/error_builder.cpp.o.d -o CMakeFiles/cpp_redis.dir/sources/builders/error_builder.cpp.o -c /home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/sources/builders/error_builder.cpp
[4/16] /usr/bin/c++  -I/home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/includes -I/home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/deps/include -I/home/particleg/vcpkg/installed/x64-linux/include -fPIC -std=c++11 -W -Wall -Wextra -O3 -g -fPIC -MD -MT CMakeFiles/cpp_redis.dir/sources/builders/integer_builder.cpp.o -MF CMakeFiles/cpp_redis.dir/sources/builders/integer_builder.cpp.o.d -o CMakeFiles/cpp_redis.dir/sources/builders/integer_builder.cpp.o -c /home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/sources/builders/integer_builder.cpp
[5/16] /usr/bin/c++  -I/home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/includes -I/home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/deps/include -I/home/particleg/vcpkg/installed/x64-linux/include -fPIC -std=c++11 -W -Wall -Wextra -O3 -g -fPIC -MD -MT CMakeFiles/cpp_redis.dir/sources/builders/simple_string_builder.cpp.o -MF CMakeFiles/cpp_redis.dir/sources/builders/simple_string_builder.cpp.o.d -o CMakeFiles/cpp_redis.dir/sources/builders/simple_string_builder.cpp.o -c /home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/sources/builders/simple_string_builder.cpp
[6/16] /usr/bin/c++  -I/home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/includes -I/home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/deps/include -I/home/particleg/vcpkg/installed/x64-linux/include -fPIC -std=c++11 -W -Wall -Wextra -O3 -g -fPIC -MD -MT CMakeFiles/cpp_redis.dir/sources/builders/bulk_string_builder.cpp.o -MF CMakeFiles/cpp_redis.dir/sources/builders/bulk_string_builder.cpp.o.d -o CMakeFiles/cpp_redis.dir/sources/builders/bulk_string_builder.cpp.o -c /home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/sources/builders/bulk_string_builder.cpp
[7/16] /usr/bin/c++  -I/home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/includes -I/home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/deps/include -I/home/particleg/vcpkg/installed/x64-linux/include -fPIC -std=c++11 -W -Wall -Wextra -O3 -g -fPIC -MD -MT CMakeFiles/cpp_redis.dir/sources/builders/array_builder.cpp.o -MF CMakeFiles/cpp_redis.dir/sources/builders/array_builder.cpp.o.d -o CMakeFiles/cpp_redis.dir/sources/builders/array_builder.cpp.o -c /home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/sources/builders/array_builder.cpp
[8/16] /usr/bin/c++  -I/home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/includes -I/home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/deps/include -I/home/particleg/vcpkg/installed/x64-linux/include -fPIC -std=c++11 -W -Wall -Wextra -O3 -g -fPIC -MD -MT CMakeFiles/cpp_redis.dir/sources/network/tcp_client.cpp.o -MF CMakeFiles/cpp_redis.dir/sources/network/tcp_client.cpp.o.d -o CMakeFiles/cpp_redis.dir/sources/network/tcp_client.cpp.o -c /home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/sources/network/tcp_client.cpp
[9/16] /usr/bin/c++  -I/home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/includes -I/home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/deps/include -I/home/particleg/vcpkg/installed/x64-linux/include -fPIC -std=c++11 -W -Wall -Wextra -O3 -g -fPIC -MD -MT CMakeFiles/cpp_redis.dir/sources/builders/reply_builder.cpp.o -MF CMakeFiles/cpp_redis.dir/sources/builders/reply_builder.cpp.o.d -o CMakeFiles/cpp_redis.dir/sources/builders/reply_builder.cpp.o -c /home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/sources/builders/reply_builder.cpp
[10/16] /usr/bin/c++  -I/home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/includes -I/home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/deps/include -I/home/particleg/vcpkg/installed/x64-linux/include -fPIC -std=c++11 -W -Wall -Wextra -O3 -g -fPIC -MD -MT CMakeFiles/cpp_redis.dir/sources/core/reply.cpp.o -MF CMakeFiles/cpp_redis.dir/sources/core/reply.cpp.o.d -o CMakeFiles/cpp_redis.dir/sources/core/reply.cpp.o -c /home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/sources/core/reply.cpp
[11/16] /usr/bin/c++  -I/home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/includes -I/home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/deps/include -I/home/particleg/vcpkg/installed/x64-linux/include -fPIC -std=c++11 -W -Wall -Wextra -O3 -g -fPIC -MD -MT CMakeFiles/cpp_redis.dir/sources/network/redis_connection.cpp.o -MF CMakeFiles/cpp_redis.dir/sources/network/redis_connection.cpp.o.d -o CMakeFiles/cpp_redis.dir/sources/network/redis_connection.cpp.o -c /home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/sources/network/redis_connection.cpp
[12/16] /usr/bin/c++  -I/home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/includes -I/home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/deps/include -I/home/particleg/vcpkg/installed/x64-linux/include -fPIC -std=c++11 -W -Wall -Wextra -O3 -g -fPIC -MD -MT CMakeFiles/cpp_redis.dir/sources/core/subscriber.cpp.o -MF CMakeFiles/cpp_redis.dir/sources/core/subscriber.cpp.o.d -o CMakeFiles/cpp_redis.dir/sources/core/subscriber.cpp.o -c /home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/sources/core/subscriber.cpp
[13/16] /usr/bin/c++  -I/home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/includes -I/home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/deps/include -I/home/particleg/vcpkg/installed/x64-linux/include -fPIC -std=c++11 -W -Wall -Wextra -O3 -g -fPIC -MD -MT CMakeFiles/cpp_redis.dir/sources/core/sentinel.cpp.o -MF CMakeFiles/cpp_redis.dir/sources/core/sentinel.cpp.o.d -o CMakeFiles/cpp_redis.dir/sources/core/sentinel.cpp.o -c /home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/sources/core/sentinel.cpp
[14/16] /usr/bin/c++  -I/home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/includes -I/home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/deps/include -I/home/particleg/vcpkg/installed/x64-linux/include -fPIC -std=c++11 -W -Wall -Wextra -O3 -g -fPIC -MD -MT CMakeFiles/cpp_redis.dir/sources/core/client.cpp.o -MF CMakeFiles/cpp_redis.dir/sources/core/client.cpp.o.d -o CMakeFiles/cpp_redis.dir/sources/core/client.cpp.o -c /home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/sources/core/client.cpp
FAILED: CMakeFiles/cpp_redis.dir/sources/core/client.cpp.o 
/usr/bin/c++  -I/home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/includes -I/home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/deps/include -I/home/particleg/vcpkg/installed/x64-linux/include -fPIC -std=c++11 -W -Wall -Wextra -O3 -g -fPIC -MD -MT CMakeFiles/cpp_redis.dir/sources/core/client.cpp.o -MF CMakeFiles/cpp_redis.dir/sources/core/client.cpp.o.d -o CMakeFiles/cpp_redis.dir/sources/core/client.cpp.o -c /home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/sources/core/client.cpp
/home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/sources/core/client.cpp: In member function ‘void cpp_redis::client::sleep_before_next_reconnect_attempt()’:
/home/particleg/vcpkg/buildtrees/cpp-redis/src/4.3.1-dddd39f2e4.clean/sources/core/client.cpp:347:21: error: ‘sleep_for’ is not a member of ‘std::this_thread’
  347 |   std::this_thread::sleep_for(std::chrono::milliseconds(m_reconnect_interval_msecs));
      |                     ^~~~~~~~~
ninja: build stopped: subcommand failed.
```

--- a/ports/cpp-redis/fix-sleep_for.patch
+++ b/ports/cpp-redis/fix-sleep_for.patch
@@ -1,0 +1,12 @@
+diff --git a/sources/core/client.cpp b/sources/core/client.cpp
+index 7ea20e2..c5d2c40 100644
+--- a/sources/core/client.cpp
++++ b/sources/core/client.cpp
+@@ -23,6 +23,7 @@
+ #include <cpp_redis/core/client.hpp>
+ #include <cpp_redis/misc/error.hpp>
+ #include <cpp_redis/misc/macro.hpp>
++#include <thread>
+ 
+ namespace cpp_redis {
+ 

--- a/ports/cpp-redis/portfile.cmake
+++ b/ports/cpp-redis/portfile.cmake
@@ -6,6 +6,8 @@ vcpkg_from_github(
     REF 4.3.1
     SHA512 abf372542c53f37f504b3211b840b100d07a8f4b2e7f5584cc7550ab16ed617838e2df79064374c7a409458d8567f4834686318ea3a40249c767e36c744c7a47
     HEAD_REF master
+    PATCHES 
+        "fix-sleep_for.patch"
 )
 
 file(COPY ${CMAKE_CURRENT_LIST_DIR}/tacopie/CMakeLists.txt DESTINATION ${SOURCE_PATH}/tacopie)

--- a/ports/cpp-redis/portfile.cmake
+++ b/ports/cpp-redis/portfile.cmake
@@ -24,14 +24,13 @@ if(VCPKG_CMAKE_SYSTEM_NAME STREQUAL "WindowsStore" OR NOT VCPKG_CMAKE_SYSTEM_NAM
     set(VCPKG_C_FLAGS_DEBUG "${VCPKG_C_FLAGS_DEBUG} -RTC1")
 endif()
 
-vcpkg_configure_cmake(
+vcpkg_cmake_configure(
     SOURCE_PATH ${SOURCE_PATH}
-    PREFER_NINJA
     OPTIONS
         -DMSVC_RUNTIME_LIBRARY_CONFIG=${MSVC_RUNTIME_LIBRARY_CONFIG}
 )
 
-vcpkg_install_cmake()
+vcpkg_cmake_install()
 
 file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
 

--- a/ports/cpp-redis/vcpkg.json
+++ b/ports/cpp-redis/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "cpp-redis",
   "version": "4.3.1",
-  "port-version": 5,
+  "port-version": 6,
   "description": "cpp-redis is a C++11 Asynchronous Multi-Platform Lightweight Redis Client, with support for synchronous operations and pipelining.",
   "homepage": "https://github.com/cpp-redis/cpp_redis",
   "license": "MIT",

--- a/ports/cpp-redis/vcpkg.json
+++ b/ports/cpp-redis/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "cpp-redis",
   "version-string": "4.3.1",
-  "port-version": 4,
+  "port-version": 5,
   "description": "cpp-redis is a C++11 Asynchronous Multi-Platform Lightweight Redis Client, with support for synchronous operations and pipelining.",
   "homepage": "https://github.com/cpp-redis/cpp_redis",
   "dependencies": [

--- a/ports/cpp-redis/vcpkg.json
+++ b/ports/cpp-redis/vcpkg.json
@@ -6,6 +6,7 @@
   "homepage": "https://github.com/cpp-redis/cpp_redis",
   "license": "MIT",
   "dependencies": [
+    "tacopie",
     {
       "name": "vcpkg-cmake",
       "host": true
@@ -13,7 +14,6 @@
     {
       "name": "vcpkg-cmake-config",
       "host": true
-    },
-    "tacopie"
+    }
   ]
 }

--- a/ports/cpp-redis/vcpkg.json
+++ b/ports/cpp-redis/vcpkg.json
@@ -1,6 +1,6 @@
 {
   "name": "cpp-redis",
-  "version-string": "4.3.1",
+  "version": "4.3.1",
   "port-version": 5,
   "description": "cpp-redis is a C++11 Asynchronous Multi-Platform Lightweight Redis Client, with support for synchronous operations and pipelining.",
   "homepage": "https://github.com/cpp-redis/cpp_redis",

--- a/ports/cpp-redis/vcpkg.json
+++ b/ports/cpp-redis/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "cpp-redis",
   "version": "4.3.1",
-  "port-version": 6,
+  "port-version": 5,
   "description": "cpp-redis is a C++11 Asynchronous Multi-Platform Lightweight Redis Client, with support for synchronous operations and pipelining.",
   "homepage": "https://github.com/cpp-redis/cpp_redis",
   "license": "MIT",

--- a/ports/cpp-redis/vcpkg.json
+++ b/ports/cpp-redis/vcpkg.json
@@ -4,7 +4,16 @@
   "port-version": 5,
   "description": "cpp-redis is a C++11 Asynchronous Multi-Platform Lightweight Redis Client, with support for synchronous operations and pipelining.",
   "homepage": "https://github.com/cpp-redis/cpp_redis",
+  "license": "MIT",
   "dependencies": [
+    {
+      "name": "vcpkg-cmake",
+      "host": true
+    },
+    {
+      "name": "vcpkg-cmake-config",
+      "host": true
+    },
     "tacopie"
   ]
 }

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -1562,7 +1562,7 @@
     },
     "cpp-redis": {
       "baseline": "4.3.1",
-      "port-version": 4
+      "port-version": 5
     },
     "cpp-taskflow": {
       "baseline": "2.6.0",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -1562,7 +1562,7 @@
     },
     "cpp-redis": {
       "baseline": "4.3.1",
-      "port-version": 6
+      "port-version": 5
     },
     "cpp-taskflow": {
       "baseline": "2.6.0",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -1562,7 +1562,7 @@
     },
     "cpp-redis": {
       "baseline": "4.3.1",
-      "port-version": 5
+      "port-version": 6
     },
     "cpp-taskflow": {
       "baseline": "2.6.0",

--- a/versions/c-/cpp-redis.json
+++ b/versions/c-/cpp-redis.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "9714d8cf0b5557d4362f54f0906295b10a154832",
+      "git-tree": "f3507ff3a30f8d0c4a4438a014c1f1e8761964d8",
       "version": "4.3.1",
       "port-version": 6
     },

--- a/versions/c-/cpp-redis.json
+++ b/versions/c-/cpp-redis.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "c8a0fd88cbd2adf8e6d501c28b0eaa3e9d3d4aa6",
+      "version-string": "4.3.1",
+      "port-version": 5
+    },
+    {
       "git-tree": "5ba5958862c17ab2f3943b0134b2ad756ae73613",
       "version-string": "4.3.1",
       "port-version": 4

--- a/versions/c-/cpp-redis.json
+++ b/versions/c-/cpp-redis.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "f3507ff3a30f8d0c4a4438a014c1f1e8761964d8",
+      "version": "4.3.1",
+      "port-version": 5
+    },
+    {
       "git-tree": "5ba5958862c17ab2f3943b0134b2ad756ae73613",
       "version-string": "4.3.1",
       "port-version": 4

--- a/versions/c-/cpp-redis.json
+++ b/versions/c-/cpp-redis.json
@@ -1,8 +1,8 @@
 {
   "versions": [
     {
-      "git-tree": "c8a0fd88cbd2adf8e6d501c28b0eaa3e9d3d4aa6",
-      "version-string": "4.3.1",
+      "git-tree": "52a89d46c40ec4646f6f7cf98f24a0281847144b",
+      "version": "4.3.1",
       "port-version": 5
     },
     {

--- a/versions/c-/cpp-redis.json
+++ b/versions/c-/cpp-redis.json
@@ -1,11 +1,6 @@
 {
   "versions": [
     {
-      "git-tree": "ec1922dba0c1f0ee807a5b6448c196dd16ee0f02",
-      "version": "4.3.1",
-      "port-version": 5
-    },
-    {
       "git-tree": "5ba5958862c17ab2f3943b0134b2ad756ae73613",
       "version-string": "4.3.1",
       "port-version": 4

--- a/versions/c-/cpp-redis.json
+++ b/versions/c-/cpp-redis.json
@@ -1,11 +1,6 @@
 {
   "versions": [
     {
-      "git-tree": "f3507ff3a30f8d0c4a4438a014c1f1e8761964d8",
-      "version": "4.3.1",
-      "port-version": 6
-    },
-    {
       "git-tree": "ec1922dba0c1f0ee807a5b6448c196dd16ee0f02",
       "version": "4.3.1",
       "port-version": 5

--- a/versions/c-/cpp-redis.json
+++ b/versions/c-/cpp-redis.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "52a89d46c40ec4646f6f7cf98f24a0281847144b",
+      "git-tree": "ec1922dba0c1f0ee807a5b6448c196dd16ee0f02",
       "version": "4.3.1",
       "port-version": 5
     },

--- a/versions/c-/cpp-redis.json
+++ b/versions/c-/cpp-redis.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "9714d8cf0b5557d4362f54f0906295b10a154832",
+      "version": "4.3.1",
+      "port-version": 6
+    },
+    {
       "git-tree": "ec1922dba0c1f0ee807a5b6448c196dd16ee0f02",
       "version": "4.3.1",
       "port-version": 5

--- a/versions/c-/cpp-redis.json
+++ b/versions/c-/cpp-redis.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "f3507ff3a30f8d0c4a4438a014c1f1e8761964d8",
+      "git-tree": "9714d8cf0b5557d4362f54f0906295b10a154832",
       "version": "4.3.1",
       "port-version": 5
     },

